force-app/main/default/lwc/rd2StatusAutomationSettings/rd2StatusAutomationSettings.js

```diff
@@ -0,0 +1,205 @@
+            ? automationSettings.numberOfDaysForClosed
+            : this.labels.commonNoneSpecific;
+        this.inputDaysForClosed = automationSettings.numberOfDaysForClosed;
+        this.maximumDaysForLapsed = (this.numberOfDaysForClosed == null)
+            ? null
+            : parseInt(this.numberOfDaysForClosed) - 1;
+    
+        this.numberOfDaysForLapsed = (automationSettings.numberOfDaysForLapsed)
+            ? automationSettings.numberOfDaysForLapsed
+            : this.labels.commonNoneSpecific
+        this.inputDaysForLapsed = automationSettings.numberOfDaysForLapsed;
+        this.minimunDaysForClosed = (this.numberOfDaysForLapsed == null)
+            ? null
+            : parseInt(this.numberOfDaysForLapsed) + 1;
+
+        this.closedStatus = automationSettings.closedStatus;
+        this.inputClosedStatus = automationSettings.closedStatus;
+
+        this.lapsedStatus = automationSettings.lapsedStatus;
+        this.inputLapsedStatus = automationSettings.lapsedStatus;
+
+        this.closedStatusOption = automationSettings.closedStatusOption;
+        this.lapsedStatusOption = automationSettings.lapsedStatusOption;
+
+        this.displayState = STATES.READY;
+    }
+
+    handleEdit() {
+        this.isEdit = true;
+    }
+
+    handleCancel() {
+        this.isEdit = false;
+    }
+
+    handleSave() {
+        this.errorMessage = null;
+        this.displayState === STATES.LOADING;
+        try {
+            saveStatusAutomationSettings(
+                { daysForLapsed: this.inputDaysForLapsed,
+                daysForClosed: this.inputDaysForClosed,
+                lapsedStatus: this.inputLapsedStatus,
+                closedStatus: this.inputClosedStatus }
+            ).then ((automationSettings) => {
+                this.assignSettings(automationSettings);
+                this.isEdit = false;
+            })
+            .catch ((error) => {
+                this.errorMessage = error.body.message;
+            })
+        } catch (error) {
+            this.errorMessge = error.body.message;
+        }
+    }
+
+    handleDaysForLapsedChanged(event) {
+        this.inputDaysForLapsed = (event.detail.value == '')
+            ? null
+            : event.detail.value;
+        this.minimunDaysForClosed = (this.inputDaysForLapsed == null)
+            ? null
+            : parseInt(this.inputDaysForLapsed) + 1;
+
+        this.validateNumberOfDays();
+    }
+
+    handleDaysForClosedChanged(event) {
+        this.inputDaysForClosed = (event.detail.value == '')
+            ? null
+            : event.detail.value;
+
+            this.maximumDaysForLapsed = (this.inputDaysForClosed == null)
+            ? null
+            : parseInt(this.inputDaysForClosed) - 1;
+
+        this.validateNumberOfDays();
+    }
+
+    validateNumberOfDays() {
+        let that = this;
+        setTimeout(() => {
+            that.template.querySelectorAll('.daysInput').forEach(element => {
```

Comment:
Instead of using a class-based selector here, can we put a `data-id` property on the element we're trying to grab?